src/main/webapp/resources/js/pages/announcement/components/AnnouncementsPage.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useState } from "react";
+import { PageWrapper } from "../../../components/page/PageWrapper";
+import {
+  getAnnouncements,
+  markAnnouncementRead,
+} from "../../../apis/announcements/announcements";
+import { Avatar, Col, Empty, Form, List, notification, Radio, Row } from "antd";
+import { PriorityFlag } from "./PriorityFlag";
+import { fromNow } from "../../../utilities/date-utilities";
+import ViewReadAnnouncement from "./ViewReadAnnouncement";
+import ViewUnreadAnnouncement from "./ViewUnreadAnnouncement";
+import { grey2 } from "../../../styles/colors";
+
+/**
+ * React component to display the page for administration of users.
+ * @returns {*}
+ * @constructor
+ */
+export function AnnouncementsPage({}) {
+  const [announcements, setAnnouncements] = useState([]);
+  const [filteredAnnouncements, setFilteredAnnouncements] = useState([]);
+  const [filter, setFilter] = useState("all");
+  const [toggleRead, setToggleRead] = useState(false);
+
+  useEffect(() => {
+    getAnnouncements()
+      .then((data) => {
+        console.log(data);
+        setAnnouncements(data);
+        setFilteredAnnouncementsOnToggle(filter, data);
+      })
+      .catch(({ message }) => {
+        notification.error({ message });
+      });
+  }, [toggleRead]);
+
+  useEffect(() => {
+    setFilteredAnnouncementsOnToggle(filter, announcements);
+  }, [filter]);
+
+  function markAnnouncementAsRead(aID) {
+    return markAnnouncementRead({ aID })
+      .then(() => {
+        setToggleRead(!toggleRead);
+      })
+      .catch(({ message }) => {
+        notification.error({ message });
+      });
+  }
+
+  function setFilteredAnnouncementsOnToggle(filterOption, announcementsList) {
+    switch (filterOption) {
+      case "read":
+        return setFilteredAnnouncements(
+          announcementsList.filter((item) => item.read)
+        );
+      case "unread":
+        return setFilteredAnnouncements(
+          announcementsList.filter((item) => !item.read)
+        );
+      default:
+        return setFilteredAnnouncements(announcementsList);
+    }
+  }
+
+  return (
+    <PageWrapper title="Announcements">
+      <Row justify="center">
+        <Col xs={24} sm={20} md={16} lg={12} xl={8}>
+          <Form>
+            <Form.Item>
```

Comment:
The radio group looks disjointed from the table, here is how to fix it:

1.  add `noStyle` to the `Form.Item` element
2.  Surround both `Row` elements in an Ant Design `Space` component.

Reply:
I can see the effects of noStyle on the Form.Item, but as soon as I wrap the Rows with a Space, it breaks the look.